### DISCUSSION TITLE
teaching | Jesse Wei

### DISCUSSION THREAD

**Discussion by giscus** [jesse-wei/jesse-wei.github.io#18] [Comments]
# teaching | Jesse Wei

I am a graduate TA for COMP 311 and was a UTA for COMP 311, 210, and 110. I am one of two awardees of the 2023 Diane Pozefsky Learning Assistant Award ($1000 prize).

https://jessewei.dev/teaching/

**Comment by anubhavelhence** [comment]
wow

**Comment by anubhavelhence** [comment]
wow
